driver/configuration/provider_viper.go

```diff
@@ -413,3 +415,11 @@ func (v *ViperProvider) ShareOAuth2Debug() bool {
 func (v *ViperProvider) PKCEEnforced() bool {
 	return viperx.GetBool(v.l, ViperKeyPKCEEnforced, false, "OAUTH2_PKCE_ENFORCED")
 }
+
+func (v *ViperProvider) LogTracingPublic() bool {
```

Comment:
I would be fine with actually having this enabled always no config required

Reply:
I fixed it!